### ISSUE TITLE
Metric specification proposal

### ISSUE THREAD

**Issue by stuartlynn** [Urban-Analytics-Technology-Platform/popgetter-cli#29]
We need some way to refer to metrics in the data request / recipe command in the CLI . We would like users to be able to specify metrics in a few different ways. Some examples might be 

this would be used like this in the CLI etc

```
popgetter data --metrics hlx:population*,8a4a71e75b02cea0aaa2ab0191504ea8f0cc8e81278e19 
```
or

```
popgetter data --hxl sdfsdf -id sdfsdfds -human_readable "Something"
```

Some suggested ways we might want to specify metrics:
```
hxl:#population+children+age0_17 // Single HXL command 
hxl:#population+children* // Range of metrics that match an HXL regex 
id:8a4a71e75b02cea0aaa2ab0191504ea8f0cc8e81278e19 // Id in the metrics table 
name:"Infants aged 0 to 4"  // Human readable name
```

```rust 
enum MetricIdSpec{
   hxl(String),
   id(String),
  ...
}
```

other ways?

**Comment by stuartlynn** [comment]
popgetter data --metrics hlx:population*,8a4a71e75b02cea0aaa2ab0191504ea8f0cc8e81278e19 --geometry municipality --with-geometry --year 2022

popgetter metrics   hxl:#population+children+age0_17 --geometry municipality --with-geometry --year 2022
popgetter geometry   municipality --year 2022

**Comment by stuartlynn** [comment]
popgetter get 
vs 
popgetter data

**Comment by stuartlynn** [comment]
This is implemented with #24
